Evaluation/Input/Content/Section2.2_Data.md

```diff
@@ -1,40 +1,19 @@
-### In vitro / physico-chemical Data <a id="invitro-and-physico-chemical-data"></a>
+### 2.2.1	In vitro / physico-chemical Data
```

Comment:
same. There are ocassionally no cross-references to this subchapter, so removing the anchor here did not create invalid links. But still: DON'T.
So here it should be then
`### 2.2.1 In vitro / physico-chemical Data <a id="invitro-and-physico-chemical-data"></a>`
Also there are too many spaces (or tab instead of space) between **2.2.1** and **In vitro / physico-chemical Data**

Reply:
Also here: too many spaces in the title after **2.2.1**

Reply:
fixed now